UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -398,7 +398,7 @@ <h3>Conformance</h3>
 						href="#conf-statements">simple statements</a> should be provided when EPUB accessibility and
 					WCAG levels are identified.</p>
 
-				<p>When there is a conformance claim to an unknown specification, the statement should indicate that Conformance to accepted standards for accessibility of this publication cannot be determined. If the publication does not include a conformance claim,
+				<p>When there is a conformance claim to an unknown specification, the statement should indicate that a    Conformance statement is present and it is to    an unknown specification. If the publication does not include a conformance claim,
```

Comment:
This comment isn't specifically about this change, but I really wonder about the usefulness of reporting unknown standards while providing no detail about them. The conformsTo metadata can reference any standard, not just an accessibility one, so what help is it to tell users the publication conforms to an unknown standard? I doubt presenting random strings in conformsTo is a good idea, so I'd argue in this case it's better to say nothing at all.

It might make more sense to say that vendors can decide for themselves whether to present conformance information they recognize but that isn't detailed in these guides, but otherwise the onus should probably be on people conforming to unrecognized standards to bring them to our attention to vet them for display.